tests/test_api.py

```diff
@@ -461,8 +461,8 @@ def test_HorizontalList(self):
     def test_ItemTemplate(self):
         assert ItemTemplate
 
-    def test_ScrollListBase(self):
-        assert ScrollListBase
+    # def test_ScrollListBase(self):
```

Comment:
We should figure out why it went missing and if it’s deliberate... 

Actually we should add tests for all the api level classes that just make sure any classes that get deleted or renamed are caught

Reply:
If I were to guess, I think when I went and added `__all__` to a bunch of modules I probably left it out assuming that it was internal to the module based on its name.

Given that the tests didn't break because they were accidentally a function instead of a class, we just never noticed.